tracer/src/Datadog.Trace/DataStreamsMonitoring/DataStreamsManager.cs

```diff
@@ -100,6 +100,22 @@ public void InjectPathwayContext<TCarrier>(PathwayContext? context, TCarrier hea
         Log.Debug("Attempted to inject null pathway context");
     }
 
+    /// <summary>
+    /// Trys to extract a <see cref="PathwayContext"/>, from the provided <paramref name="headers"/>
+    /// Or as a fallback from  <paramref name="binaryHeaders"/>
+    /// If data streams is disabled, or no pathway is present, returns null.
+    /// </summary>
+    public PathwayContext? ExtractLegacyPathwayContext(IBinaryHeadersCollection binaryHeaders, IHeadersCollection headers)
```

Comment:
nit-picky, but I would expect the API to have the fallback last
```suggestion
    public PathwayContext? ExtractLegacyPathwayContext(IHeadersCollection headers, IBinaryHeadersCollection binaryHeaders)
```